bam/build.gradle

```diff
@@ -0,0 +1,40 @@
+plugins {
+    id 'com.android.library'
+    id 'org.jetbrains.kotlin.android'
+}
+
+android {
+    namespace 'com.localmotive.bam'
```

Comment:
Why namespace does not match app's package?

Are you planning to have a separate package name?